client/reader/following/use-subscriptions.ts

```diff
@@ -0,0 +1,42 @@
+import { SubscriptionManager } from '@automattic/data-stores';
+import { useMemo, useEffect } from 'react';
+
+export function useSubscriptions() {
+	const { data: subscriptionsCount, isLoading: isLoadingCount } =
+		SubscriptionManager.useSubscriptionsCountQuery();
+	const {
+		data: siteSubscriptions,
+		isLoading: isLoadingSiteSubscriptions,
+		refetch: refetchSiteSubscriptions,
+	} = SubscriptionManager.useSiteSubscriptionsQuery();
+
+	const isLoading = isLoadingCount || isLoadingSiteSubscriptions;
+	const blogCount = subscriptionsCount?.blogs ?? 0;
+
+	const hasNonSelfSubscriptions = useMemo( () => {
+		if ( blogCount === 0 ) {
+			return false;
+		}
+
+		// If we have site subscriptions data, filter out self-owned blogs.
+		if ( siteSubscriptions?.subscriptions ) {
+			const nonSelfSubscriptions = siteSubscriptions.subscriptions.filter(
+				( sub ) => ! sub.is_owner
+			);
+			return nonSelfSubscriptions.length > 0;
+		}
+
+		return blogCount > 0;
```

Comment:
Seems like we can simplify it.

```suggestion
		return true;
```

Reply:
Done in https://github.com/Automattic/wp-calypso/pull/97030/commits/a9a533ca86b75b45601f7c12b83be29ce170e6e4